### PR TITLE
updated bundle and added schemas and service

### PR DESCRIPTION
**Pull request by smisidjan**

### CODE REVIEW

File: Installation/Schema/vrijbrp.document.schema.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "$id": "https://vrijbrp.nl/schemas/vrijbrp.document.schema.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
```
Comment (WilcoLouwerse) [inline Installation/Schema/vrijbrp.document.schema.json]: Wrong $schema, please use the newest version of this

File: Installation/Schema/vrijbrp.eersteInschrijving.schema.json [hunk 2]
```diff
@@ -0,0 +1,102 @@
+{
+    "$id": "https://vrijbrp.nl/schemas/vrijbrp.eersteInschrijving.schema.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
```
Comment (WilcoLouwerse) [inline Installation/Schema/vrijbrp.eersteInschrijving.schema.json]: Wrong $schema, please use the newest version of this

File: Installation/Schema/vrijbrp.importRecord.schema.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
+    "$id": "https://vrijbrp.nl/schemas/vrijbrp.importRecord.schema.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
```
Comment (WilcoLouwerse) [inline Installation/Schema/vrijbrp.importRecord.schema.json]: Wrong $schema, please use the newest version of this

File: src/ActionHandler/FirstRegistrationHandler.php [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    /**
+     * Returns the required configuration as a https://json-schema.org array.
+     *
+     * @return array The configuration that this  action should comply to
+     */
+    public function getConfiguration(): array
+    {
+        return [
+            '$id'         => 'https://vrijbrp.nl/vrijbrp.zaak.handler.json',
+            '$schema'     => 'https://json-schema.org/draft/2020-12/schema',
```
Comment (WilcoLouwerse) [inline src/ActionHandler/FirstRegistrationHandler.php]: Wrong $schema, please use the newest version of this

Comment (WilcoLouwerse) [reply src/ActionHandler/FirstRegistrationHandler.php]: not schema but ActionHandler as well

**Review by rjzondervan — APPROVED** [review]

File: src/Service/FirstRegistrationService.php [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+    // public function firstRegistrationHandler(array $data, array $configuration): array
+    // {
+    // $this->data          = $data;
+    // $this->configuration = $configuration;
+    //
+    // $this->logger->debug("FirstRegistrationService -> firstRegistrationHandler()");
+    //
+    // return ['response' => 'Hello. Your EersteInschrijvingBundle works'];
+    //
+    // }//end petStoreHandler()
```
Comment (WilcoLouwerse) [inline src/Service/FirstRegistrationService.php]: Commented code with petStore in it ?

**Review by WilcoLouwerse — CHANGES_REQUESTED** [review]

**Review by WilcoLouwerse — APPROVED** [review]

**Pull request merged**